pkgs/top-level/all-packages.nix

```diff
@@ -12689,6 +12689,16 @@ with pkgs;
 
   upstart-check-config = callPackage ../os-specific/linux/upstart/check-config.nix {};
 
+  usbguard = usbguard_libgcrypt;
+
+  usbguard_libgcrypt = libsForQt5.callPackage ../os-specific/linux/usbguard {
+    libsodium = null;
```

Comment:
What is the advantage for having both versions in nixpkgs? Do both variants provide the same features?